lib/ec2ssh/ec2_instances.rb

```diff
@@ -1,22 +1,69 @@
-require 'aws-sdk-v1'
+require 'aws-sdk'
 
 module Ec2ssh
   class Ec2Instances
     attr_reader :ec2s, :aws_keys
 
-    def initialize(aws_keys, regions)
+    class InstanceWrapper
+      class TagsWrapper
+        def initialize(tags)
+          @tags = tags
+        end
+
+        # simulate
+        def [](key)
+          if key.is_a? ::String
+            warn '`tags[String]` syntax is deleted. Please upgrade your .ec2ssh syntax.'
+          end
+          super
+        end
+
+        private
+
+        def method_missing(name, *args, &block)
+          @tags.public_send(name, *args, &block)
+        end
+
+        def respond_to_missing?(symbol, include_private)
+          @tags.respond_to?(symbol, include_private)
+        end
```

Comment:
This causes exception under the following condition:
- Some EC2 instance with tags exists
- And `tags['Foo']` called in the dotfile

Because in that case `@tags` might be an `Array` and it doesn't have `String` keys.

Reply:
That's right.
In this case, I expect to print warning and raise a exception.
Should I fallback to `tag('Foo') if `tags['Foo']` called in dotfile?

Reply:
I changed at dbc327d87c6788329c6cc591e6d9cf169e2dbb38.